### PR TITLE
WIP: [Evo] ref #420 - Complete the profile during app authorizations

### PR DESCRIPTION
**Pull request by franck-boucher**

### CODE REVIEW

File: src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package org.oasis_eu.portal.front.my;
+
+import org.oasis_eu.portal.front.generic.PortalController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+/**
+ * User: schambon
```
Comment (bobeal) [inline src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java]: is it really your name ?

File: src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * Date: 6/11/14
+ */
+@Controller
+@RequestMapping("/sp")
```
Comment (bobeal) [inline src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java]: this sp (for single page) bothers me. we need to find a more appropriate name.

File: src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+public class SinglePageOzwilloController extends PortalController {
```
Comment (bobeal) [inline src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java]: no need to add Ozwillo in the name, we know where we are.

File: src/main/resources/public/js/pages/profile.js [hunk 4]
```diff
@@ -353,6 +422,17 @@ class PasswordAccount extends React.Component {
     }
 }
 
+const ConditionnalClaimsField = ({ children, voluntaryClaims = [], essentialClaims = [], field }) =>
```
Comment (bobeal) [inline src/main/resources/public/js/pages/profile.js]: s/Conditionnal/Conditional

File: src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java (hunk 2 above)
Comment (bobeal) [reply src/main/java/org/oasis_eu/portal/front/my/SinglePageOzwilloController.java]: /popup (or /standalone)

File: src/main/resources/public/js/pages/profile.js [hunk 5]
```diff
@@ -90,6 +91,9 @@ class Profile extends React.Component {
 
     render() {
         const userProfile = this.state.userProfile;
+        const searchParams = new URLSearchParams(this.props.location.search)
```
Comment (bobeal) [inline src/main/resources/public/js/pages/profile.js]: beware of https://caniuse.com/#feat=urlsearchparams

**Comment by franck-boucher** [comment]
Recreated directly on Ozwillo repo:
https://github.com/ozwillo/ozwillo-portal/pull/422

**Pull request closed without merging**